runtime/litentry/src/weights/mod.rs

```diff
@@ -17,7 +17,7 @@
 #![allow(clippy::unnecessary_cast)]
 
 pub mod cumulus_pallet_xcmp_queue;
-pub mod frame_system;
+// pub mod frame_system;
```

Comment:
Why disable system weight?

Reply:
Because it has a different interface and needs to be re-benchmarked.

Having it uncommented will cause compilation error